client/src/components/pages/createProfile/style.css

```diff
@@ -0,0 +1,22 @@
+.create-profile--header {
+  width: 70%;
+  text-align: center;
+  margin: 0 auto;
+}
+
+.ant-steps-item-process .ant-steps-item-icon {
+  background: var(--primary-color, pink);
+  border-color: var(--primary-color, pink);
+}
+
+.ant-steps-item-finish .ant-steps-item-icon {
```

Comment:
no space between classes in css, if there spaces the second class not effective

Reply:
see the comment above